### ISSUE TITLE
TypeError: null is not an object (evaluating 'constr.toString().match(self.rx_function)')

### ISSUE THREAD

**Issue by mauro-ferrario** [hbi99/defiant.js#118]
Hi, I have this issue only on some old version of iPad. The issue is on line 458:

cnName = constr.toString().match(self.rx_function)[1];

I don't know if it's the best solution, but I solved in that way:

cnName = constr.toString().match(self.rx_function) ? [1] : null;

**Comment by hbi99** [comment]
Is there any way to reproduce this without the old iPad?

**Comment by mauro-ferrario** [comment]
No, I just noticed this issue on old iPad (iPad 3), on chrome simulator or other iOS devices is fine.
